osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/src/main/java/org/opensmartgridplatform/adapter/protocol/iec60870/domain/services/ClientConnectionService.java

```diff
@@ -21,6 +22,24 @@
      */
     ClientConnection getConnection(RequestMetadata requestMetadata) throws ConnectionFailureException;
 
+    /**
+     * Closes the {@link Connection}, sends a disconnect request and closes the
+     * socket.
+     *
+     * @param deviceIdentification
+     *            Device for which to close the connection.
+     */
+    void disconnect(String deviceIdentification);
```

Comment:
Hmm, I think you **_disconnect_** a device by **_closing_** the connnection, so for a device it would be appropriate to use disconnect?
Anyway, renamed methods to:
closeConnection(String deviceIdentification)
close(ClientConnection connection)

Reply:
OK with the rename, maybe if the javadocs had looked more like your explanation, I wouldn't have commented about this.